### PR TITLE
Add gradle ui

### PR DESCRIPTION
**Pull request by iainduncani**
Fixes for issue #107, issue #108 and issue #109

---
This change is [Reviewable](https://reviewable.io/reviews/wasdev/tool.accelerate.core/118)

### CODE REVIEW

**Comment by iainduncani** [comment]
This should soon be available to view at:
http://app-accel-staging.mybluemix.net/start/

**Comment by katheris** [comment]
Reviewed 10 of 12 files at r1, 2 of 2 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---

*[liberty-starter-application/src/main/webapp/js/services/appacc.js, line 219 at r2](https://reviewable.io:443/reviews/wasdev/tool.accelerate.core/118#-KZ6bDkZFwi9Fzu8UrAv:-KZ6bDkZFwi9Fzu8UrAw:b-h5twkb) ([raw file](https://github.com/wasdev/tool.accelerate.core/blob/f6cc94d260b8124301eef58a06c75f34568ac792/liberty-starter-application/src/main/webapp/js/services/appacc.js#L219)):*
> ```JavaScript
>         for (var validBuildType in buildType) {
>           if (buildType.hasOwnProperty(validBuildType)) {
>             if (newBuildType === validBuildType) {
> ```

Why do we need both the for loop and this if( buildType.hasOwnProperty(validBuildType)) ?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/wasdev/tool.accelerate.core/118)*

**Comment by iainduncani** [comment]
*[liberty-starter-application/src/main/webapp/js/services/appacc.js, line 219 at r2](https://reviewable.io:443/reviews/wasdev/tool.accelerate.core/118#-KZ6bDkZFwi9Fzu8UrAv:-KZ6gEOuJiTzYxWbCDyi:b84cy46) ([raw file](https://github.com/wasdev/tool.accelerate.core/blob/f6cc94d260b8124301eef58a06c75f34568ac792/liberty-starter-application/src/main/webapp/js/services/appacc.js#L219)):*
<details><summary><i>Previously, katheris (Katherine Stanley) wrote…</i></summary><blockquote>

Why do we need both the for loop and this if( buildType.hasOwnProperty(validBuildType)) ?
</blockquote></details>

Because stack overflow said so ;-) 
http://stackoverflow.com/questions/8312459/iterate-through-object-properties
basically because "in buildType" will return all the properties on buildType including ones inherited from the base JavaScript object and I wanted to restrict it to just properties declared in the object in this file.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/wasdev/tool.accelerate.core/118)*
